pixano/models/inference_model.py

```diff
@@ -62,16 +62,20 @@ def __init__(
     def create_table(
         self,
         process_type: str,
+        ann_type: str,
         views: list[str],
         dataset: Dataset,
     ) -> DatasetTable:
         """Create inference table in dataset
 
         Args:
             process_type (str): Process type
-                                - 'obj' for preannotation
-                                - 'segment_emb' for segmentation embedding precomputing
-                                - 'search_emb' for semantic search embedding precomputing
+                                - 'ann' for annotations
+                                - 'segment_emb' for segmentation embeddings
+                                - 'search_emb' for semantic search embeddings
+            ann_type (str): Annotation type
+                                - 'Pre-annotation' for pre-annotations to accept or reject as Ground Truth
+                                - 'Model run' for annotations to compare to Ground Truth
```

Comment:
Don't you think it would be easier to have 4 process types instead of 3 process_types and 2 annotation_type ?
process_type (str): Process type
  - 'pre_ann' for preannotations
  - 'model_run' for model run annotations
  - 'segment_emb' for segmentation embeddings
  - 'search_emb' for semantic search embeddings

Reply:
Yes, I should have seen that. Merged `ann_type` with `process_type` in 48eba9c44f6681ca5295da59142572dda6eebf0a.